adrs/php/0002_module_structure.md

```diff
@@ -0,0 +1,17 @@
+# Module structure
+
+## Motivation
+
+Standardize the modular design of the different modules
+
+## Decision
+
+The modules have a similar structure between each other. At the root of a module you can find: 
+
+- Facade: it's the entry point of the module. 
+  If another module wants to interact with another module, they will do it via the Facade.
```

Comment:
```md
If one module wants to interact ...
```
I find it more "natural". What do you think? 😄

Reply:
Right, thanks